applications/tile-inspector/src/decodePbf.js

```diff
@@ -0,0 +1,17 @@
+// Use this tool to convert a protobuf tile to an 4wings intArray
+
+import Protobuf from 'pbf'
+
+const decodePbf = (data) => {
```

Comment:
I'd move this to the `data-transforms` package to ensure we use the same that we are using on the tests of fourwings-aggregate